src/simsopt/geo/curve.py

```diff
@@ -841,14 +937,227 @@ def wrap(data):
         ppl = np.asarray([c.gamma().shape[0] for c in curves])
     data = np.concatenate([i*np.ones((ppl[i], )) for i in range(len(curves))])
     pointData = {'idx': data}
-
     if extra_data is not None:
         pointData = {**pointData, **extra_data}
 
     polyLinesToVTK(str(filename), x, y, z, pointsPerLine=ppl, pointData=pointData)
 
 
-def create_equally_spaced_curves(ncurves, nfp, stellsym, R0=1.0, R1=0.5, order=6, numquadpoints=None):
+def setup_uniform_grid(s, s_inner, s_outer, Nx, Ny, Nz, Nmin_factor=2.5):
```

Comment:
suggest renaming from `setup_uniform_grid` because this function does more than just return a uniform grid of points